### PR TITLE
shop-sdk v2 beta

### PR DESCRIPTION
**Pull request by codeliger**
I replaced all colorwaySKU with colorwaySizeAssetSKU

### CODE REVIEW

File: src/init.ts [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+		console.error(error);
+	});
+
+	let fittingRoom: FittingRoom = {
+		user: undefined,
+		shop: undefined,
+		firebase: InitFirebase(),
+		manager: InitModalManager(modalDivID),
+
+		onSignout(colorwaySKU: string) {
```
Comment (edalford11) [inline src/init.ts:39]: colorwaySizeAssetSKU

File: src/init.ts [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+
+		onClose() {
+			this.manager.Close();
+		},
+
+		onNavBack() {
+			window.history.back();
+		},
+
+		onTryOn(colorwaySKU) {
```
Comment (edalford11) [inline src/init.ts:57]: colorwaySizeAssetSKU

missing string type?

File: src/init.ts [hunk 3]
```diff
@@ -0,0 +1,297 @@
...
+						error: L.SomethingWentWrong,
+						onClose: this.onClose.bind(this),
+						onNavBack: this.onNavBack
+					}));
+					this.framesCallback(error);
+				}
+			});
+		},
+
+		afterSignIn(user, colorwaySKU) {
```
Comment (edalford11) [inline src/init.ts:128]: colorwaySIzeAssetSKU

Should these also have types?

File: src/init.ts [hunk 4]
```diff
@@ -0,0 +1,297 @@
...
+				console.error('catch uierror', error);
+				this.manager.Open(ErrorModal({
+					error: error.message,
+					onClose: this.onClose.bind(this),
+					onNavBack: this.onNavBack
+				}));
+			});
+		},
+
+		onSignIn(colorwaySKU) {
```
Comment (edalford11) [inline src/init.ts:175]: colorwaySizeAssetSKU

string type?

File: src/init.ts [hunk 5]
```diff
@@ -0,0 +1,297 @@
...
+					this.user = u;
+					this.afterSignIn(this.user, colorwaySKU);
+					this.manager.Close();
+				}).catch((UIError) => {
+					validationError(UIError.message);
+				});
+			};
+		},
+
+		onNavSignIn(colorwaySKU: string) {
```
Comment (edalford11) [inline src/init.ts:192]: colorwaySizeAssetSKU

File: src/init.ts [hunk 6]
```diff
@@ -0,0 +1,297 @@
...
+				this.manager.Open(SignInModal({
+					email,
+					onSignIn: this.onSignIn(colorwaySKU),
+					onNavForgotPassword: this.onNavForgotPassword(colorwaySKU),
+					onNavScanCode: this.onNavScanCode.bind(this)
+				}));
+			};
+		},
+
+		onPasswordReset(colorwaySKU: string) {
```
Comment (edalford11) [inline src/init.ts:204]: colorwaySizeAsserSKU

File: src/init.ts [hunk 7]
```diff
@@ -0,0 +1,297 @@
...
+					this.manager.Open(ErrorModal({
+						error: UIError.message,
+						onClose: this.onClose.bind(this),
+						onNavBack: this.onNavBack
+					}));
+				});
+			};
+		},
+
+		onNavForgotPassword(colorwaySKU: string): (email: string) => void {
```
Comment (edalford11) [inline src/init.ts:221]: colorwaySIzeAssetSKU

File: src/init.ts [hunk 8]
```diff
@@ -0,0 +1,297 @@
...
+					onPasswordReset: this.onPasswordReset(colorwaySKU)
+				}));
+			};
+		},
+
+		onNavScanCode(): void {
+			this.manager.Open(ScanCodeModal({}));
+		},
+
+		TryOn(colorwaySKU: string, callback: (frames: types.TryOnFrames | Error) => void) {
```
Comment (edalford11) [inline src/init.ts:235]: colorwaySizeAssetSKU

**Review by edalford11 — CHANGES_REQUESTED** [review]

**Comment by codeliger** [comment]
I replaced all colorwaySKU with colorwaySizeAssetSKU

**Review by edalford11 — APPROVED** [review]

**Pull request merged**